tests/ctst/features/iam-policies/IAMUser.feature

```diff
@@ -55,8 +55,8 @@ Feature: IAM Policies for IAM Users
       | GetObject      | Allow  | iu-go-bucket3-1/*        | iu-go-bucket3 | exists         |            |
       | GetObject      | Allow  | iu-go-bucket4/go-object1 | iu-go-bucket4 | exists         | go-object  |
       | GetObject      | Deny   | *                        | iu-go-bucket5 | exists         |            |
-      | GetObject      | Deny   | iu-go-bucket5/*          | iu-go-bucket5 | exists         |            |
-      | GetObject      | Deny   | iu-go-bucket6/go-object  | iu-go-bucket6 | exists         | go-object  |
+      | GetObject      | Deny   | iu-go-bucket6/*          | iu-go-bucket6 | exists         |            |
```

Comment:
Should be great to be able to pass variables generated during the test in the examples, so that the `given an existing bucket ...` will create an bucket named randomly and in the examples we put something like {bucketName} and it takes the one generated in the beginning
Not sure if it is feasable tho..

Reply:
We do that in most scenarios, but here we need to specify the resource name, and we actually want to have control over it because sometimes we alter it, so we want to keep it...

Reply:
I know we do that in most scenario, what I suggest here is to be able to use it in example, so here for example we could do `${bucketName}/*` in the ressource name
But ofc we need to keep support for creating static bucket names